Source/PeleC.cpp

```diff
@@ -348,6 +366,8 @@ PeleC::PeleC()
     mms_src_evaluated(false)
 #endif
 {
+  typical_values_chem.resize(NUM_SPECIES + 1);
```

Comment:
There is a subtle difference. typical_values_chem is local to each level. My thought was to have different set of typvals for each level, the logic being a flame may well resolved in a finer level while it may be washed out at a coarser one which means the typvals are different? I removed it as a member now, it is a local varaible